lib/conjur/base.rb

```diff
@@ -202,8 +225,29 @@ def refresh_token
         end
       end
 
-      def update_token_born
-        self.token_born = gettime
+      def gettime
+        monotonic_time
+      end
```

Comment:
If the `gettime` method is the same in both `APIKeyAuthenticator` and `LocalAuthenticator`, why not include it in the TokenExpiration module like you did with `update_token_born`? Why even have a method for this - you could just replace calls to `gettime` with `monotonic_time`